symplyphysics/laws/astronomy/large_semi_axis_of_orbit.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+from sympy.physics.units import gravitational_constant
+
+# Description
+## Let the body move in an elliptical orbit. Then its large semi-axis depends on the mass of the body around which it
+## rotates and the orbital velocity.
+
+## Law is: a = G * M / v^2, where
+## a - large semi-axis of the orbit,
+## G - gravitational constant,
+## M - mass of the body around which the movement takes place,
+## v - orbital velocity.
+
+large_semi_axis = Symbol("large_semi_axis", units.length)
```

Comment:
I suggest to update https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/gravity/orbital_velocity_from_length_of_large_half_axis_and_planet_mass.py to have the same names of variables. Also you can move that law to `astronomy`